packages/transfer-app/src/SendBalance.tsx

```diff
@@ -2,8 +2,8 @@
 // This software may be modified and distributed under the terms
 // of the Apache-2.0 license. See the LICENSE file for details.
 
-import { ApiContext } from '@polkadot/ui-api';
-import { AddressSummary, Grid, Header, Icon, Input, MarginTop, NavButton, Stacked } from '@polkadot/ui-components';
+import { ApiContext } from '@substrate/ui-api';
```

Comment:
I still want to pull the context stuff into the `polkadot-js/ui` repo (`@polkadot/ui-api` - have that package anyway) - since I have those and you have those as well, although it is for different types of APIs. Will take a peek at what you have in context and try to align.

In short, continue as-is, but would like to make this stuff usable in a single place as well. (Having said that, while in dev, it is easier having it inside the repo, so wouldn't worry too much at this point)

Reply:
Sounds good, I think that's a good idea.

So I would put:
- ApiContext
- The `<Subscribe />` components
- HOC

The combination of these 3 can give end-users the choice how they want to connect the api to their react app